.gitignore

```diff
@@ -2,6 +2,4 @@ martin_test
 memory_eater
 memload
 memuse
-pig
```

Comment:
Did you mean to do this?

Reply:
Yes.  That was preventing us from pushing any changes within the pig directory.